ding/policy/ppg.py

```diff
@@ -34,6 +34,585 @@ def create_shuffled_dataloader(data, batch_size):
     return DataLoader(ds, batch_size=batch_size, shuffle=True)
 
 
+@POLICY_REGISTRY.register('ppg_onpolicy')
+class PPGOnPolicy(Policy):
```

Comment:
rename these two classes, `PPGPolicy` and `PPGOffPolicy` , and modify related parts